examples/passing/RecordUnionDupe.purs

```diff
@@ -0,0 +1,12 @@
+module Main where
+
+import Prelude
+import Data.Record (merge)
+import Control.Monad.Eff.Console (log, CONSOLE)
+import Control.Monad.Eff (Eff)
+import Unsafe.Coerce (unsafeCoerce)
+
+main :: Eff (console::CONSOLE) Unit
+main = do
+  let result = (merge {left:1} {left:"Done"}).left
+  log $ unsafeCoerce result
```

Comment:
Well if the type of `merge` is `RowUnion l r o => Record l -> Record r -> Record o`, then we really only have only possible implementation, at least if `RowUnion` means a union which allows duplicates. The meaning of a duplicate label in row `x` in `Record x` is "take the leftmost type", so the merge on records has to be left-biased.